### PR TITLE
create video status endpoint

### PR DESCRIPTION
**Pull request by jamesgorrie**
# Get video info endpoint

We currently need to know whether videos are expired as we don't have this information available from flex when the video is embedded. 

I could have used the [video json](http://gu.com/film/video/2016/mar/01/sacha-baron-cohan-appears-as-ali-g-at-oscars-video.json) as it has the `"status": "GONE"` but the file is a few kilobytes big, and this call might happen across multiple videos on a page.

There is also a question around caching, but it seems the media endpoints don't have much in place for that? Is this correct, or shall we be changing that? @sndrs?
## What does this change?

Nothing, it adds value.
## What is the value of this and can you measure success?

We will have an endpoint, it serves JSON, valuable JSON.
## Does this affect other platforms - Amp, Apps, etc?

Nope, but we eventually be doing something similar there.
## @akash1810 

_Does this PR meet the [contributing guidelines](https://github.com/guardian/frontend/blob/issue_pr_templates/.github/CONTRIBUTING.md#submission)?_

### CODE REVIEW

File: dev-build/conf/routes [hunk 1]
```diff
@@ -411,6 +411,7 @@ GET            /$path<[\w\d-]*(/[\w\d-]*)?/gallery/.*>
 GET            /$path<[\w\d-]*(/[\w\d-]*)?/(cartoon|graphic|picture)/.*>.json                                                    controllers.ImageContentController.renderJson(path)
 GET            /$path<[\w\d-]*(/[\w\d-]*)?/(cartoon|graphic|picture)/.*>                                                         controllers.ImageContentController.render(path)
```
Comment (gtrufitt) [inline dev-build/conf/routes]: Think you also need to update standalone routes

**Comment by akash1810** [comment]
:+1:

File: dev-build/conf/routes (hunk 1 above)
Comment (jamesgorrie) [inline dev-build/conf/routes]: What do you mean?

Comment (gtrufitt) [inline dev-build/conf/routes]: This file: https://github.com/guardian/frontend/blob/master/standalone/conf/standalone.routes#L139 Do you need to add your new route there? IIRC standalone is used for preview or something - @rich-nguyen ?

Comment (rich-nguyen) [inline dev-build/conf/routes]: Correct, @gtrufitt

Comment (jamesgorrie) [inline dev-build/conf/routes]: What is that file for?

Comment (jamesgorrie) [inline dev-build/conf/routes]: Just reread it, sorry, for preview.

**Pull request merged**